packages/zoe/test/swingsetTests/zoe-metering/testBuiltins.js

```diff
@@ -1,13 +1,13 @@
 import harden from '@agoric/harden';
 
-export const makeContract = zoe => {
-  const invite = zoe.makeInvitation(() => {}, 'tester');
-  return harden({
-    invite,
-    publicAPI: {
+export const makeContract = zcf => {
+  const invite = zcf.makeInvitation(() => {}, 'tester');
+  zcf.initPublicAPI(
+    harden({
```

Comment:
FYI, you may experience a trivial merge conflict here (with the new-SES branch), the `1e7` had to change to `1e8` in a nearby line